workflow/scripts/phenotype_process/eval_features.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+
+from lib.phenotype_process.eval_features import plot_feature_heatmap
+
+
+# Load SBS processing files
+phenotype_cp_min = pd.read_hdf(snakemake.input[0])
+
+# Generate and save feature heatmaps
+min_feature_names = [col for col in phenotype_cp_min.columns if col.endswith("_min")]
+for feature_name in min_feature_names:
+    # Generate feature heatmap evaluation
+    df_summary_one, fig = plot_feature_heatmap(
+        phenotype_cp_min, feature=feature_name, shape="6W_sbs", return_summary=True
```

Comment:
Should be ph layout

Reply:
Done in https://github.com/cheeseman-lab/brieflow/pull/19/commits/4f9b5573489c309e6959f983cc37532b05cba341